### PR TITLE
Added trend link in the navbar

### PR DESCRIPTION
**Pull request by rahmlad-aramide**
I've added Trend link and it's now getting displayed on both the Desktop navbar or Mobile bottom navbar, below are the screenshots of how it's now:
trend-after
trend-after-desktop

### CODE REVIEW

File: components/core/Sidebar.tsx [hunk 1]
```diff
@@ -30,6 +31,12 @@ const Sidebar = ({ children }: { children: React.ReactNode }) => {
         active: pathname === "/",
         href: "/",
       },
+      {
+        icon: BiTrendingUp,
+        label: "Trend",
+        href: "/trend",
```
Comment (p7uverma) [inline components/core/Sidebar.tsx:37]: trend is not a page wirte /trends

**Review by p7uverma — CHANGES_REQUESTED** [review]

**Comment by p7uverma** [comment]
@rahmlad-aramide Please fix the url.

**Comment by rahmlad-aramide** [comment]
> @rahmlad-aramide Please fix the url.

Or is it suppose to be linked to another url which is not `trend`, if yes kindly provide me the correct one.

**Comment by rahmlad-aramide** [comment]
> @rahmlad-aramide Please fix the url.

going to do that now

**Comment by rahmlad-aramide** [comment]
Check now, I've fixed it @p7uverma

**Review by p7uverma — APPROVED** [review]

**Pull request merged**